wordpress-mobile/WordPress-Android
# Jetpack: Display multisite not supported screen for multisites

Fixes #14946

This PR displays multisite not supported message for a multisite. It uses the standard `ActionableEmptyView` to be consistent with other error states UI on the Scan screen.

Corresponding FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2069

Light | Dark
-----|-----
![multi_lt_1](https://user-images.githubusercontent.com/1405144/126782981-bbd600e9-6c1a-4672-9723-309a4b1113b0.png) | ![multi_dark_1](https://user-images.githubusercontent.com/1405144/126783293-bbd5d78f-d185-43ce-a048-183464991253.png)

To test:

1. Open the Jurassic Ninja site: https://jurassic.ninja/specialops/
2. Pick a Jetpack plan (use Jetpack Complete).
3. Launch a Jetpack Mutlisite on subdomains site.
4. Connect to this site and open the app.
5. Go to: My Site -> Scan
6. Verify that you see multisite not supported message

**Merge Instructions**

1. Make sure corresponding FluxC PR is merged
2. Wait till I add a FluxC tag and update it in build.gradle
3. Remove "Not Ready for Merge" label
4. Merge the PR

## Regression Notes
1. Potential unintended areas of impact
Changes have no impact on the rest of the functionality as this is a separate flow applicable only for multi sites.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A (added unit tests for the functionality added)

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
